Common/UnitDefinitions/Force.json

```diff
@@ -142,8 +156,10 @@
     {
       "SingularName": "OunceForce",
       "PluralName": "OunceForce",
-      "FromUnitToBaseFunc": "{x} * 2.780138509537812e-1",
-      "FromBaseToUnitFunc": "{x} / 2.780138509537812e-1",
+      "FromUnitToBaseFunc": "{x} * 0.27801385095378125",
+      "FromBaseToUnitFunc": "{x} / 0.27801385095378125",
+      "XmlDocSummary": "An ounce-force is 1⁄16 of a pound-force, or about 0.2780139 newtons.",
```

Comment:
We can divide the function in pound-force by 16 here to preserve as much precision as possible:
`"FromUnitToBaseFunc": "{x} * 4.4482216152605 / 16"`